### PR TITLE
Initial tabular classification Santander

### PR DESCRIPTION
**Pull request by PabloAMC**
# Before submitting

- [x] Was this discussed/approved via a Github issue? (no need for typos and docs improvements)
- [x] Did you make sure to update the docs?
- [x] Did you write any new necessary tests?

## What does this PR do?

Fixes # (issue).

## PR review

Anyone in the community is free to review the PR once the tests have passed.
If we didn't discuss your PR in Github issues there's a high chance it will not be merged.

## Did you have fun?

Make sure you had fun coding 🙃

### CODE REVIEW

**Review by Borda — DISMISSED** [review]
could you pls provide metafile with required details :rabbit:

**Review by Borda — COMMENTED** [review]
hi, seems you are accidentally changing some other files in Flash turorials

**Comment by PabloAMC** [comment]
> hi, seems you are accidentally changing some other files in Flash turorials

I changed the name of the electricity_forecasting folder to tabular_forecasting. I believe that might be much more informative.

File: flash_tutorials/tabular_classification/.meta.yml [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+  This tutorial covers using Lightning Flash to predict whether some users will make a transaction based on numerical tabular data. Since it is an inbalanced dataset, we use SMOTE to balance the dataset. We then use native TabNet to train the model.
+requirements:
+  - pandas==1.1.5
+  - lightning-flash[tabular]>=0.6.0
+  - torchmetrics
+  - imblearn
+  - sklearn
+accelerator:
+  - GPU
+  - CPU
```
Comment (Borda) [inline flash_tutorials/tabular_classification/.meta.yml:20]:
```suggestion
  - CPU
datasets:
  kaggle:
    - santander-customer-transaction-prediction
```
missing data: https://www.kaggle.com/c/santander-customer-transaction-prediction

File: flash_tutorials/tabular_classification/santander.py [hunk 2]
```diff
@@ -0,0 +1,158 @@
...
+
+seed_everything(seed=42)
+
+# %% [markdown]
+# ## Loading and inspecting the data
+
+# %%
+# ## Download the data
+data_path = os.environ.get("PATH_DATASETS", "_datasets")
+path_santander = os.path.join(data_path, "santander")
```
Comment (Borda) [inline flash_tutorials/tabular_classification/santander.py:40]:
```suggestion
path_santander = os.path.join(data_path, "santander-customer-transaction-prediction")
```

**Review by Borda — COMMENTED** [review]
seems that you included unrelated images from the tutorial

**Comment by PabloAMC** [comment]
Why do we get error
```
python -m pytest flash_tutorials/tabular_classification/santander.ipynb -v --nbval
deactivate
rm -rf flash_tutorials/tabular_classification/venv
401 - Unauthorized
Error: Process completed with exit code 1.
```
?

**Review by Borda — CHANGES_REQUESTED** [review]
pls, remove images that are not related to this tutorial...

**Comment by Borda** [comment]
@PabloAMC seems like some variables in the notebook are not defined... mind checking the failing test?

**Comment by Borda** [comment]
we may not promote Flash examples as it was archived some time ago, apology nd thank you for all you help :rabbit:

**Pull request closed without merging**